src/query-builder/returning-interface.ts

```diff
@@ -75,4 +75,6 @@ export interface ReturningInterface<DB, TB extends keyof DB, O> {
    * that support `returning` such as PostgreSQL.
    */
   returningAll(): ReturningInterface<DB, TB, Selectable<DB[TB]>>
+
+  returningAll<T extends TB>(table: T): ReturningInterface<DB, TB, Selectable<DB[T]>>
```

Comment:
Good call! But it's premature. 

Let's not add this now since it doesn't make sense in all query builders.

Reply:
Removed. Makes sense.